packages/node-sdk/src/client.ts

```diff
@@ -845,8 +849,10 @@ export class BucketClient {
       context.company = { id: companyId };
     }
 
-    const contextWithTracking = { ...context, enableTracking: true };
```

Comment:
Why split then then build again later?

Reply:
good catch. Reverting this. Happened just because of back/forth code churn.